tasks/config/project/config.js

```diff
@@ -137,11 +133,11 @@ module.exports = {
         config.globs.components = componentsExceptIndividuals.length === 1 ? componentsExceptIndividuals[0] : '{' + componentsExceptIndividuals.join(',') + '}';
 
         // components that should be built, but excluded from main .css/.js files
-        config.globs.individuals = individuals.length === 1 ? individuals[0] : (individuals.length > 1)
-            ? '{' + individuals.join(',') + '}'
-            : undefined;
-
-        return config;
```

Comment:
done remove the return, it's needed at 2 calls in other files

Reply:
in the problematic/fixed ones :)